pkg/virt-controller/watch/migration.go

```diff
@@ -659,6 +657,11 @@ func (c *MigrationController) handleSignalMigrationAbort(migration *virtv1.Virtu
 	return nil
 }
 
+func (c *MigrationController) isMigrationProtected(pdb *v1beta1.PodDisruptionBudget) bool {
+	return pdb.Spec.MinAvailable.IntValue() == int(pdb.Status.DesiredHealthy) &&
+		pdb.Generation == pdb.Status.ObservedGeneration
+}
```

Comment:
what's guaranteeing that the MinAvailable is set to `2` here instead of `1`?

Reply:
because we always deal with the expanded PDB, and if we reached this line that means `minAvailable` == 2  (otherwise an error would have been thrown at line 722). I can make it explicit if you think it's better.

Reply:
>  I can make it explicit if you think it's better.

yeah, make it explicit. The reasoning here is that even though what you have now is technically safe, i could see this becoming an issue during a refactor later on.